src/Components/ADT3DSceneBuilder/Internal/Behaviors/Widgets/WidgetBuilders/DataHistoryWidgetBuilder/Internal/TimeSeriesFormCallout.tsx

```diff
@@ -152,9 +161,21 @@ const TimeSeriesFormCallout: React.FC<IProp> = ({
                         expression: seriesToEdit?.expression
                     }}
                     onChange={handlePropertyChange}
-                    allowedPropertyValueTypes={numericPropertyValueTypes}
+                    allowedPropertyValueTypes={
+                        allowedTimeseriesPropertyValueTypes
+                    }
                     required
                 />
+                {seriesToEdit?.propertyType &&
```

Comment:
can we push this message through the `description` field of the `modeledPropertyBuilder`? It will then get styled appropriately as if it was attached to the dropdown and not just a plain text block.

Reply:
ok :)